client/src/data/locales/es-ES.json

```
@@ -2,5 +2,25 @@
   "header.maps": "MAPAS",
   "header.database": "BASE DE DATOS DE LICENCIAS",
   "header.resources": "RECURSOS DE LICENCIAS",
-  "header.home": "INICIOS"
-}
+  "header.home": "INICIOS",
+  "footer.license": "LICENCIA",
+  "footer.toSucceed": "PARA TENER ÉXITO",
+  "footer.disclaimerTitle": "Descargo de responsabilidad:",
+  "footer.disclaimerText": "La información proporcionada en este sitio web es solo para fines informativos generales. Los datos representados en este sitio no están garantizados como precisos o completos. La información puede cambiar sin previo aviso.",
+  "footer.builtBy": "Creado por Code for Boston para Offsite Hospitality",
+  "footer.copyright": "©, OFFSITE LLC Todos los derechos reservados",
+  "footer.siteMap": "Mapa del sitio",
+  "footer.homepage": "Inicio",
+  "footer.map": "Mapa",
+  "footer.database": "Base de datos",
+  "footer.resources": "Recursos importantes",
+  "footer.importantLinks": "Enlaces importantes",
+  "footer.bostonBoard": "Junta de Licencias de Boston",
+  "footer.abcc": "Comisión de Bebidas Alcohólicas",
+  "footer.application": "Solicitud de Licencia de Boston",
+  "footer.analyzeBoston": "Analizar Boston",
+  "footer.moreFromUs": "Más de nosotros",
+  "footer.offsite": "Offsite Hospitality",
+  "footer.bootcamp": "Bootcamp de gestión de bares",
+  "footer.codeForBoston": "Code for Boston"
```

We are holding off on translations until copy is finalized ^^
```suggestion
  "header.home": "INICIOS"
```